docs/installation/getting-started/quick-start.md

```diff
@@ -0,0 +1,315 @@
+</info-box>
+
+<info-box warning>
+	Note that in this guide, the editor class is used directly (i.e. we use `@ckeditor/ckeditor5-editor-classic` instead of `@ckeditor/ckeditor5-build-classic`).
+
+	No {@link installation/advanced/predefined-builds editor builds} are used, because adding new plugins to these requires rebuilding them anyway. This can be done by {@link installation/getting-started/installing-plugins customizing a build} or by including CKEditor 5 source into your application (like in this guide).
+</info-box>
+
+```js
+// app.js
+
+import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
+import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ Essentials, Paragraph, Bold, Italic ],
+		toolbar: [ 'bold', 'italic' ]
+	} )
+	.then( editor => {
+		console.log( 'Editor was initialized', editor );
+	} )
+	.catch( error => {
+		console.error( error.stack );
+	} );
+```
+
+You can now run webpack to build the application. To do that, call the `webpack` executable:
+
+```bash
+./node_modules/.bin/webpack --mode development
+```
+
+You can also install `webpack-cli` globally (using `npm install -g`) and run it via a globally available `webpack`.
+
+Alternatively, you can add it as an [npm script](https://docs.npmjs.com/misc/scripts):
+
+```js
+"scripts": {
+	"build": "webpack --mode development"
+}
+```
+
+And use it with:
+
+```bash
+yarn run build
+```
+
+npm adds `./node_modules/.bin/` to the `PATH` automatically, so in this case you do not need to install `webpack-cli` globally.
+
+<info-box>
+	Use `webpack --mode production` if you want to build a minified and optimized application. Learn more about it in the [webpack documentation](https://webpack.js.org/concepts/mode/).
+
+	**Note:** Prior to version 1.2.7, `uglifyjs-webpack-plugin` (the default minifier used by webpack) had a bug which caused webpack to crash with the following error: `TypeError: Assignment to constant variable.`. If you experienced this error, make sure that your `node_modules` contain an up-to-date version of this package (and that webpack uses this version).
+
+	**Note:** CKEditor 5 Builds use [`Terser`](https://github.com/terser/terser) instead of `uglifyjs-webpack-plugin` because [the later one seems to be unsupported anymore](https://github.com/ckeditor/ckeditor5/issues/1353).
+</info-box>
+
+If everything worked correctly, you should see:
+
+```
+p@m /workspace/quick-start> ./node_modules/.bin/webpack --mode development
+Hash: c96beab038124d61568f
+Version: webpack 4.15.1
+Time: 3023ms
+Built at: 2018-07-05 17:37:38
```

Comment:
Is this what you will see in 2022?

Reply:
> Version: webpack 4.15.1

Make sure to review webpack and other dependencies because of the PR downgraded versions. See https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/quick-start.html#lets-start.